lib/std/crypto/aegis.zig

```diff
@@ -169,12 +170,15 @@ fn Aegis128LGeneric(comptime tag_bits: u9) type {
             tag.* = state.mac(tag_bits, ad.len, m.len);
         }
 
-        /// m: message: output buffer should be of size c.len
-        /// c: ciphertext
-        /// tag: authentication tag
-        /// ad: Associated Data
-        /// npub: public nonce
-        /// k: private key
+        /// `m`: Message
+        /// `c`: Ciphertext
+        /// `tag`: Authentication tag
+        /// `ad`: Associated Data
+        /// `npub`: Public nonce
+        /// `k`: Private key
+        /// Asserts `c.len == m.len`.
+        ///
+        /// Contents of `m` are undefined if an error is returned.
```

Comment:
English is not my native language, but `Contents are undefined` sounds weird. Why not `Content is undefined`?

Reply:
For what it's worth, both of those sound fine to a native English speaker.

Reply:
Thanks for your input, Andrew!